packages/eip-5792-middleware/src/constants.ts

```diff
@@ -18,3 +30,47 @@ export enum EIP5792ErrorCode {
   UnknownBundleId = 5730,
   RejectedUpgrade = 5750,
 }
+
+// wallet_getCallStatus
+export enum GetCallsStatusCode {
+  PENDING = 100,
+  CONFIRMED = 200,
+  FAILED_OFFCHAIN = 400,
+  REVERTED = 500,
+  REVERTED_PARTIAL = 600,
+}
+
+export const GetCallsStatusStruct = tuple([StrictHexStruct]);
+
+// wallet_getCapabilities
+export const GetCapabilitiesStruct = tuple([
+  HexChecksumAddressStruct,
+  optional(array(StrictHexStruct)),
+]);
+
+export const CapabilitiesStruct = record(
+  string(),
+  type({
+    optional: optional(boolean()),
+  }),
+);
+
+// wallet_sendCalls
+export const SendCallsStruct = tuple([
+  object({
+    version: nonempty(string()),
+    id: optional(StrictHexStruct),
+    from: optional(HexChecksumAddressStruct),
+    chainId: StrictHexStruct,
+    atomicRequired: boolean(),
+    calls: array(
+      object({
+        to: optional(HexChecksumAddressStruct),
+        data: optional(StrictHexStruct),
+        value: optional(StrictHexStruct),
+        capabilities: optional(CapabilitiesStruct),
+      }),
+    ),
+    capabilities: optional(CapabilitiesStruct),
+  }),
+]);
```

Comment:
i know some of these aren't actually typescript types, but they're effectively runtime types and probably belong in `types.ts`. What are your thoughts here?